Documentation/ceph-backup.md

```diff
@@ -0,0 +1,483 @@
+---
+title: Back Up
+weight: 2200
```

Comment:
The weight indicates in what order the topics should be listed in the table of contents. This value will add it just before the "Ceph Tools" --> "Advanced Configuration".
```suggestion
weight: 11250
```

Reply:
Ah okey, I did not understand what value should be placed. Now I do, thank you!